src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -41,8 +41,12 @@
         'list.language': 'Language'|trans({}, 'Admin.Global'),
         'list.example': 'Example'|trans({}, 'Admin.Global'),
         'list.edit': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
-        'list.reset': 'Reset settings'|trans({}, 'Admin.Actions'),
-        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature')
+        'list.reset': 'Reset'|trans({}, 'Admin.Actions'),
+        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature'),
+        'modal.restore.title': 'Are you sure you want to restore default settings?'|trans({}, 'Admin.International.Feature'),
+        'modal.restore.apply': 'Restore'|trans({}, 'Admin.Actions'),
+        'modal.restore.cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+        'modal.restore.body': 'Restoring your currencies’ default settings will delete all the customizations you made before. Parameters will look just the same as when a currency is freshly imported.'|trans({}, 'Admin.International.Feature')
```

Comment:
The same body message is used for the general restore button (which restore default settings AND customizations for all the languages) and the specific one (which restores ONLY customization for ONE language)
The title can be the same it doesn't shock me, however I think the body message should be adapted for those two use cases to avoid misunderstanding.
@LouiseBonnard do you have an idea what messages we could use to differentiate both use cases?